eq_translations/extractable_strings.py

```diff
@@ -136,6 +136,27 @@
         "context": "Question",
         "additional_context": ["ListHeading", "ListDescription"],
     },
+    {
+        "json_path": "$..question.definition.title",
+        "description": "Question definition heading",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..question.definition.contents[*].title",
+        "description": "Question definition heading",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..question.definition.contents[*].description",
+        "description": "Question definition description",
+        "context": "Question",
+    },
+    {
+        "json_path": "$..question.definition.contents[*].list[*]",
```

Comment:
Do we need to add `"json_path": "$..question.definition.contents[*].content"` to extractable string?

Reply:
Content has been removed so that it is consistent with other things like guidance etc